### DISCUSSION TITLE
Debug file

### DISCUSSION THREAD

**Discussion by Nc2meu2** [RainbowMiner/RainbowMiner#2049] [Q&A]
Hello everyone,

I've been getting lots of crashes on my system lately. I installed 2 new 1660 TI and my computer works fine until I start RBM. I have a current debug file and I removed the 1660 TI from my systems until I find out what's going on. Any suggestions will be appreciated. 

Ps, I increased the allocated memory, the drivers are up to date, windows 10 reads all my gpus with no issues, powershell is updated and I'm running RainbowMiner v4.8.3.3 with auto update enabled. 

My Q is, should I post this as an issue...or where in GitHub.

**Comment by RainbowMiner** [comment]
It's ok here, just upload your Debug file. One thing to look at is your total power consumption. Is/Are your PSU(s) big enough? If it/they are too small, your system will crash. Also, if you are using riser cards, try to interchange them to be sure all cards work all right.

**Reply by Nc2meu2** [reply]
Thank you for the fast response. I currently have 3 psu (1000, 750, 600) I had the 1000 set up to power 2 3070, 1 1660ti gpus, and the motherboard. 750 to power 1 1080 and 1 3070ti. 600 to power 2 1660ti. Not using any Risers. The Gpus are connected directly to the motherboard and each one is powered individually. Will post the file in a sec.

**Comment by Nc2meu2** [comment]
[debug_2022-04-21.zip](https://github.com/RainbowMiner/RainbowMiner/files/8534106/debug_2022-04-21.zip)
